projects/go-lib/src/lib/components/go-action-sheet/go-panel/go-panel.component.scss

```diff
@@ -61,13 +61,9 @@ $theme-light-transparentized: rgba($theme-light-color, .5);
       color: $ui-color-negative;
     }
 
-    .go-panel {
-      &__title {
-        & &-text {
+    .go-panel__title-text {
```

Comment:
This is probably just a Visual Studio Code vs GitHub issue, but please make sure line 64 is indented properly.

Reply:
Indentation has been corrected